src/info_command.c

```diff
@@ -74,6 +74,12 @@ static void renderIndexDefinitions(RedisModule_Reply *reply, IndexSpec *sp) {
     REPLY_KVSTR_SAFE("payload_field", rule->payload_field);
   }
 
+  if (rule->index_all) {
+    REPLY_KVSTR_SAFE("indexes_all", "true");
+  } else {
+    REPLY_KVSTR_SAFE("indexes_all", "false");
```

Comment:
maybe use the same strings you used for the input arguments.
i.e enable and disable
just to be consistent

Reply:
Sounds good

Reply:
Thing is that this needs to be lower case (to be consistent with the rest of the response), so this can become `index_all` to more resemble the creation argument format. Then it doesn't really matter anymore. But I can go with `index_all`..